common/changes/@autorest/core/patch-1_2021-03-14-18-45.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@autorest/core",
+      "comment": "",
+      "type": "none"
```

Comment:
```suggestion
      "type": "patch"
```

Reply:
Interesting type of workflow. Thanks for helping me with that.

Reply:
yeah its using rush monorepo tool which use those change files to genrate teh changelogs when releasing